src/components/YourCompany/YourCompany.vue

```diff
@@ -285,19 +309,25 @@ export default class YourCompany extends Mixins(CommonMixin, DateMixin, LegalApi
     return `${this.getBusinessNumber || '[Incorporation Number]'} B.C. Ltd.`
   }
 
+  /** Name Request applicant info */
   private get nrApplicant (): NameRequestApplicantIF {
     return this.getNameRequest?.applicant
   }
 
+  /** Name Request expiry */
   private get expiryDate (): string {
     return new Date(this.getNameRequest.expiry).toDateString()
   }
 
-  /** The recognition (aka effective) datetime. */
+  /** The recognition/founding (aka effective) datetime. */
   private get recognitionDateTime (): string {
-    return this.getOriginalEffectiveDate
-      ? (this.convertUtcTimeToLocalTime(this.getOriginalEffectiveDate.toString()) + ' Pacific Time')
-      : 'Unknown'
+    return this.isCorrection()
+      ? this.getOriginalEffectiveDate
+        ? (this.convertUtcTimeToLocalTime(this.getOriginalEffectiveDate.toString()) + ' Pacific Time')
+        : 'Unknown'
+      : this.getBusinessFoundingDate
+        ? (this.convertUtcTimeToLocalTime(this.getBusinessFoundingDate.toString()) + ' Pacific Time')
+        : 'Unknown'
```

Comment:
I recommend you refactor this to an if-then-else construct.

Ternary expressions are nice only when they're simple :)

Reply:
lol okie doke, i don't disagree. Will update!

Reply:
Updated in Next commit!